frontend/src/static/js/v_summary.js

```diff
@@ -207,7 +229,14 @@ window.vSummary = {
       }
 
       addHash('timeframe', this.filterTimeFrame);
-      addHash('mergegroup', this.isMergeGroup);
+
+      let mergedGroupsMapHash = Object.keys(this.mergedGroupsMap)
+          .filter((key) => this.mergedGroupsMap[key])
+          .join(window.HASH_DELIMITER);
+      if (mergedGroupsMapHash.length === 0) {
+        mergedGroupsMapHash = 'none';
```

Comment:
I guess this can be an empty string `''`, to follow the way on how checkedFileTypes in authorship tab is done.